test/issues/issues-test.js

```diff
@@ -520,4 +520,17 @@ describe("issues", function () {
             stub.restore();
         });
     });
+
+    describe("#1900 - calledWith returns false positive", function () {
+        it("should return false when call args don't match", function () {
+            var stub = sinon.stub();
```

Comment:
In order to eliminate potential for regressions, it would be better to verify this using `spy`, as a `stub` is a `spy`, but a `spy` is not a `stub` (the same is true for `spy` and `fake`).

Something like this:

```js
var spy = sinon.spy();
var dateOne = new Date("2018-07-01");
var dateTwo = new Date("2018-07-31");

spy(dateOne);

var calledWith = spy.calledWith(dateTwo);
assert.same(calledWith, false);
```